### PR TITLE
Breaking: #97135 - web-ts -  Removed support for module handling based on TBE_M…

### PR DESCRIPTION
**Pull request by linawolf**
…ODULES_EXT

References https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/3 
Releases: main, 12.4

### CODE REVIEW

**Comment by linawolf** [comment]
the provided code for migration does not seem to work currently: https://forge.typo3.org/issues/103627. Should we make a note?

File: Documentation/PageTsconfig/Mod/WebTs.rst [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
-   items can be found by browsing *System > Configuration > $GLOBALS['TBE_MODULES_EXT']*.
+..  _pageblindingfunctionmenuoptions-webts-migration:
 
-   .. figure:: /Images/ManualScreenshots/Template/FunctionMenuTemplateModule.png
-      :alt: The function menu of the Template module
+Migration from menu.function to options.hideModules
+---------------------------------------------------
 
-      The function menu of the Template module
+..  todo: link to options.hideModules once it is documented
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebTs.rst:33]: hideModules is documented, it is user TSconfig:
https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/UserTsconfig/Options.html#hidemodules

File: Documentation/PageTsconfig/Mod/WebTs.rst [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
-
-      # Disable the item "Template Analyzer"
-      mod.web_ts.menu.function {
-         TYPO3\CMS\Tstemplate\Controller\TemplateAnalyzerModuleFunctionController = 0
-      }
+    # after
+    options.hideModules := addToList(web_typoscript_analyzer)
 
+You can find the names of all TypoScript modules in
+:file:`EXT:tstemplate/Configuration/Backend/Modules.php`.
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebTs.rst:46]: Please use t3src for easy access.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

**Comment by linawolf** [comment]
Will be continued in https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/pull/463

**Pull request closed without merging**